Proxy/faraday_uart_stack/layer_4_service.py

```diff
@@ -129,7 +132,8 @@ def receive_datagram_queue_hasitem(self):
 
     def process_received_datagram(self, datagram):
         parsed_datagram_dict = layer_4_protocol.parse_packet(datagram)
-        #print "RX'd:", parsed_datagram_dict
+        if DEBUG:
```

Comment:
In this case `parsed_datagram_dict` is an unused variable because the print statement was commented out. I added the `DEBUG` flag so that we could keep the print statement around but in a disabled state.